packages/render-fidelity-tools/src/components/renderers/babylon-viewer.ts

```diff
@@ -98,7 +98,10 @@ export class BabylonViewer extends LitElement {
         'Camera',
         alpha,
         beta,
-        orbit.radius,
+        Math.max(
+            orbit.radius,
+            0.0001),  // TODO: babylon doesn't support 0 radius for now, and
```

Comment:
i just used an arbitrary small value. If you don't like this value, im happy to change